sickbeard/webserve.py

```diff
@@ -1738,6 +1742,10 @@ def setStatus(self, show=None, eps=None, status=None, direct=False):
 
                     trakt_data.append((epObj.season, epObj.episode))
 
+                    if int(status) == SKIPPED and epObj.torrent_hash != '':
```

Comment:
the point here is: if you change status of episode to default, you have to remove torrent from transmission and clean torrent_hash